### ISSUE TITLE
no meta file generated after the training

### ISSUE THREAD

**Issue by sachinkmohan** [divamgupta/image-segmentation-keras#360]
There are no meta file generated. These are the files generated after 2 epochs. Can someone please help.

```
vgg_unet_1.00001.index
vgg_unet_1.00001.data-00000-of-00001
vgg_unet_1.00002.index
vgg_unet_1.00002.data-00000-of-00001
```

**Comment by varungupta31** [comment]
Did you find a solution?
In my case, there is only `checkpoint`  `_config.json`
and checkpoint contains

```
model_checkpoint_path: ".00001"
all_model_checkpoint_paths: ".00001"
```

**Comment by sachinkmohan** [comment]
I am using model.save or model.save_weights. So I am no more working with the checkpoint files. However it would be good to know about this.

**Comment by varungupta31** [comment]
Yes, I too then changed the code to save an '.h5' file, and now my model gets saved. (refered [this](https://stackoverflow.com/questions/70828564/tf-keras-callbacks-modelcheckpoint-callback-is-always-overwritten-despite-of-set#:~:text=from%20tensorflow.keras%20import%20callbacks%0A%0Acallbacks.ModelCheckpoint(%0A%20%20%20%20%20filepath%3D%27model.%7Bepoch%3A02d%7D%2D%7Bval_loss%3A.4f%7D.h5%27%2C%20%0A%20%20%20%20%20save_freq%3D%27epoch%27%2C%20verbose%3D1%2C%20monitor%3D%27val_loss%27%2C%20%0A%20%20%20%20%20save_weights_only%3DTrue%2C%20%0A) ))
In your case, I atleast see index and data files (assuming you made no changed in the cloned repo). However, the code doesn't generate even these in my case.

Thanks for your inputs.
